File: solutions/javascript/DailyTemperatures/DailyTemperatures.js
/**
 * @param {number[]} temperatures
 * @return {number[]}
 */
var dailyTemperatures = function(temperatures) {
    let res = [];
    for (let i = 0; i < temperatures.length - 1; i++) {
        for (let j = i + 1; j < temperatures.length; j++) {
            if (temperatures[j] > temperatures[i]) {
                res[i] = j - i;
                break;
            }
        }
        if (res[i] === undefined) {
            res[i] = 0;
        }
    }
    res.push(0);
    return res;
};

File: solutions/java/DailyTemperatures/DailyTemperatures.java
class Solution {
    public int[] dailyTemperatures(int[] T) {
        for (int i = 0; i < T.length; i++) {
            int tmp = T[i];
            T[i] = 0;
            for (int j = i + 1; j < T.length; j++) {
                if (T[j] > tmp) {
                    T[i] = j - i;
                    break;
                }
            }
        }
        return T;
    }
}